presto-jdbc/src/main/java/io/prestosql/jdbc/ConnectionProperties.java

```diff
@@ -67,6 +68,7 @@
             .add(KERBEROS_KEYTAB_PATH)
             .add(KERBEROS_CREDENTIAL_CACHE_PATH)
             .add(ACCESS_TOKEN)
+            .add(EXTRA_CREDENTIALS)
```

Comment:
Can please add this to JDBC documentation?

Reply:
In `presto-docs/src/main/sphinx/installation/jdbc.rst`